both/routes/user.js

```diff
@@ -30,6 +30,8 @@ Router.map(function () {
     template: 'userEdit',
     waitOn: function () {
       Meteor.subscribe('users');
+      Meteor.subscribe('userClasses');
```

Comment:
What is the difference between userClasses and classes? There should only be one collection.